src/pagerduty.js

```diff
@@ -275,11 +278,54 @@ function processSchedulesFromConfig(done) {
   });
 }
 
+function getEngineersOncall(scheduleId, escalationPolicyId, done) {
```

Comment:
I think that can have better name what about `getOncallForHandover` ?

Reply:
how about `getEngineersOncallForHandover`? as this returns the users (plus their relevant oncalls)